README.md

```diff
@@ -1 +1,37 @@
 # workbench
+
+## Getting the code
+
+* Run "git clone https://https://github.com/all-of-us/workbench"
+
+To make changes, do:
+
+* git checkout master
+* git pull
+* git checkout -b USERNAME/BRANCH_NAME
+* (make changes and git add / commit them)
+* git push -u origin USERNAME/BRANCH_NAME
+
+And make a pull request in your browser at 
+https://github.com/all-of-us/workbench based on your upload.
+
+After responding to changes, merge in GitHub.
+
+
+## Setup
+
+* Download Gradle 4.0 from https://services.gradle.org/distributions/gradle-4.0-bin.zip
+
+* Unzip it and add bin/gradle to the path.
+
+* Run tools/setup_env.sh.
```

Comment:
What do you think of switching to Python for helper scripts? I think it could be a big improvement. Reusable, testable, stack traces for easier debugging. Calling subprocess.check_call all the time is kinda verbose, but I think it's worth it for the gains.